CMakeLists.txt

```diff
@@ -756,6 +756,12 @@ endif()
 
 if(UNIX AND NOT APPLE AND NOT ANDROID)
   set(HANDSHAKER_ARGS "-handshaker-path" $<TARGET_FILE:handshaker>)
+  if(DEFINED ENV{AWS_LC_RUNNER_START_INDEX})
```

Comment:
Would it make sense to name the `AWS_LC_RUNNER_START_INDEX` and `AWS_LC_RUNNER_END_INDEX` environment variables more "SSL" specific? Like `AWS_LC_SSL_RUNNER_START_INDEX` and `AWS_LC_SSL_RUNNER_END_INDEX`

Reply:
Addressed.